lib/build.js

```diff
@@ -266,10 +285,29 @@ export default {
 
       let stdout = '';
       let stderr = '';
+      let liveErroMatchQueue = '';
+      let liveErroMatchHandle = target.liveErrorMatch ? function (d) {
```

Comment:
Why is this conditioned by the target? Any reason to not alway do live error matching?